app/views.py

```diff
@@ -78,6 +78,80 @@ def admin():
     return render_template("admin.html", vehicles=vehicles, services=services)
 
 
+@app.route("/admin/services")
+def admin_services_management():
+    services = services_model.all()
+    return render_template("admin/services/index.html", services=services)
+
+
+@app.route("/admin/services/adding", methods=["GET", "POST"])
+def admin_services_adding():
+    if request.method == "POST":
```

Comment:
instead of splitting the logic here based on the method type, can we create a separate method for saving content when the method us POST, and limit this one to only GET methods and returning services 

`def admin_services_save` this will be used for both inserting or updating new content